tests/integration/test_source_git.py

```diff
@@ -459,6 +460,18 @@ def test_srpm_merge_storm(
     }
 
 
+def test_srpm_merge_storm_dirty(api_instance_source_git):
+    ref = "0.1.0"
+    sg_path = Path(api_instance_source_git.upstream_local_project.working_dir)
+    mock_spec_download_remote_s(sg_path, sg_path / "fedora", ref)
+    create_merge_commit_in_source_git(sg_path, go_nuts=True)
+    (sg_path / "malt").write_text("Mordor\n")
+    with pytest.raises(PackitException) as ex:
+        with cwd(sg_path):
+            api_instance_source_git.create_srpm(upstream_ref=ref)
+    assert "he source-git repo is dirty" in str(ex.value)
```

Comment:
```suggestion
    assert "The source-git repo is dirty" in str(ex.value)
```

Reply:
my copy-pasta skills are pretty poor, thanks for spotting this